<launch>
  <node name="stoplight_watcher" pkg="iarrc" type="stoplight_watcher_py.py" output="screen">
      <param name="stoplight_topic" type="string" value="/start_detected" />
      <param name="img_topic" type="string" value="/camera/image_raw" />
  </node>
</launch>
